cogs/login.py

```diff
@@ -0,0 +1,146 @@
+import logging
+from typing import Any, Dict, cast
+
+import discord
+from discord.ui import Select, View
+from discord import Color, Embed, Guild
+from discord.ext import commands
+from discord.ext.commands.context import Context
+
+
+from main import ManChanBot
+
+from .commandbase import CommandBase
+
+import json
+
+with open("login.json") as f:
+        config = json.load(f)
+
+class Login(CommandBase):
+    @commands.command()
+    async def login(self, ctx: Context):
+        # Create Embed with Python, reads from json config file
+        embed=Embed()
+        embed.title = "Various Login Information"
+        embed.description = "Please do not share this information with anyone else!"
+        embed.set_footer(text="For Mobile Friendly, copy the password using the Selection Menu below")
+        embed.color = Color.blue()
+
+        embed.add_field(
```

Comment:
Already spoke to you, but we wanted to generalize the login info and not hard code it. If we ever change/remove/add a login we would have to make a new ticket or restart the bot.

If all this info is in a json, then it will dynamically update the embed.

Easy fix though, you already have the code.

It would pretty much be something like:

```
for key,value in json.items():
  embed.add_field(...)
```